src/isBefore.ts

```diff
@@ -1,15 +1,15 @@
 import { date } from "./date"
-import type { DateInput } from "./types"
+import type { DateInput, MaybeDateInput } from "./types"
 
 /**
- * Is the first date before the second one?
+ * Is the first date before the second one or `now`?
  *
  * @param inputDate - The date that should be before the other one to return true
- * @param dateToCompare - The date to compare with
+ * @param dateToCompare - The date to compare with or `now` if nothing given
```

Comment:
in JSDocs, an optional param should be wrapped in square bracket (see [JSDocs](https://jsdoc.app/tags-param)). Also to be really aligned with JSDocs, it the type should come before the param name, so it should be something like 

```ts
* @param {DateInput} inputDate - The date that should be before the other one to return true
* @param {MaybeDateInput} [dateToCompare] - The date to compare with or `now` if nothing given
```

Reply:
yeah we could at use square brackets to indicate a param is optional. although we don't need to add types because we use typescript to provide the types